spec/rails_helper.rb

```diff
@@ -9,11 +9,10 @@
 # Add additional requires below this line. Rails is not loaded until this point!
 require 'support/factory_bot'
 require 'cocina/rspec'
-require 'capybara/rspec'
```

Comment:
Moved all Capybara stuff into its own support file so the Rails helper is less messy.